ckanext/canada/templates/internal/home/quick_links.html

```diff
@@ -25,49 +30,214 @@ <h2 class="h5"><a href="{{
+            <a class="wb-lbx" href="#new_od"><i class="fa fa-question-circle-o" aria-hidden="true"></i></a>
+            <section id="new_od" class="mfp-hide modal-dialog modal-content overlay-def">
+              <header class="modal-header">
+                <h2 class="modal-title">{{ _('Create an Open Data Record') }}</h2>
+              </header>
+              <div class="modal-body">
+                {{ _('Add data about Government of Canada services, financials or national demographic information that is relevant to Canadians.') }}
+              </div>
+            </section>
+          </li>
+          {{ quick_link('experiment') }}
+          {{ quick_link('nap') }}
+          {{ quick_link('inventory') }}
+        </ul>
       </section>
-      <section>
-        <h2 class="h5"><a href="{{ h.url_for('info_new') }}">{{ _('Open Information Records') }}</a></h2>
-        <p>{{ _('Add information about government programs, activities and publications.') }}</p>
+      <section id="ql-open-info">
+        <h2 class="h5">{{ _('Open Information') }}</h2>
+        <ul class="list-unstyled">
+          <li>
+            <a href="{{ h.url_for('info_new') }}"><i class="fa fa-folder-open mrgn-rght-sm"></i>{{ _('Create Information Asset') }}</a>
+            <a class="wb-lbx" href="#new_oi"><i class="fa fa-question-circle-o" aria-hidden="true"></i></a>
+            <section id="new_oi" class="mfp-hide modal-dialog modal-content overlay-def">
+              <header class="modal-header">
+                <h2 class="modal-title">{{ _('Create Information Asset') }}</h2>
+              </header>
+              <div class="modal-body">
+                {{ _('Add information about government programs, activities and publications.') }}
+              </div>
+            </section>
+          </li>
+        </ul>
+      </section>
+<!--      <section id="ql-sug-datasets">-->
+<!--        <h2 class="h5">{{ _('Suggested Datasets') }}</h2>-->
+<!--        <ul class="list-unstyled">-->
+<!--          <li>-->
+<!--            <a href="{{ h.url_for('dataset_search', portal_type='prop') }}"><i class="fa fa-comment mrgn-rght-sm"></i>{{ _('Update Status') }}</a>-->
+<!--            <a class="wb-lbx" href="#sug_dataset"><i class="fa fa-question-circle-o" aria-hidden="true"></i></a>-->
+<!--            <section id="sug_dataset" class="mfp-hide modal-dialog modal-content overlay-def">-->
+<!--              <header class="modal-header">-->
+<!--                <h2 class="modal-title">{{ _('Suggested Datasets - Update Status') }}</h2>-->
+<!--              </header>-->
+<!--              <div class="modal-body">-->
+<!--                {{ _('Access the suggested datasets that your organization has received in order to provide a status update.') }}-->
+<!--              </div>-->
+<!--            </section>-->
+<!--          </li>-->
+<!--        </ul>-->
+<!--      </section>-->
```

Comment:
please remove this instead of commenting it out in the HTML

Reply:
This is temporary, until the suggested datasets work is complete. You still prefer that it gets removed?